pkg/app/server/grpcapi/grpcapi.go

```diff
@@ -191,6 +191,21 @@ func getEncriptionKey(se *model.Piped_SecretEncryption) ([]byte, error) {
 	}
 }
 
+func gRPCErrorForEntityOperation(err error, msg string) error {
```

Comment:
nits
```suggestion
func gRPCEntityOperationError(err error, msg string) error {
```
this name is more natural.

Reply:
Nice. Applied it! 👍